CIScripts/Test/PesterHelpers/PesterHelpers.ps1

```diff
@@ -0,0 +1,67 @@
+﻿Import-Module Pester
+
+function Consistently {
+    <#
+    .SYNOPSIS
+    Utility wrapper for making sure that the assert is consistently true. It works by retrying
+    the assert every Interval seconds, up to Duration.
+    .PARAMETER ScriptBlock
+    ScriptBlock containing a Pester assertion.
+    .PARAMETER Interval
+    Interval (in seconds) between retries.
+    .PARAMETER Duration
+    Total time to spend between retries.
+    #>
+    Param (
+        [Parameter(Mandatory=$true)] [ScriptBlock] $ScriptBlock,
+        [Parameter(Mandatory=$false)] [int] $Interval = 1,
+        [Parameter(Mandatory=$false)] [int] $Duration = 3
```

Comment:
Perhaps make `Duration` mandatory?

Reply:
I'm not sure, it can be unnecessary boilerplate.

Reply:
This function is not used very often (compared to eventually), so I'm not sure about providing default.

Reply:
Why would you like to make `Duration` mandatory?